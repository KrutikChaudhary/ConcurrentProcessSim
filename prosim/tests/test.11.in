2 3
Proc1 2 1
DOOP 5
HALT

Proc2 2 1
DOOP 3
HALT
