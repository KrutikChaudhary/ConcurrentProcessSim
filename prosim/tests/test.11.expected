00000: process 1 new
00000: process 1 ready
00000: process 2 new
00000: process 2 ready
00000: process 1 running
00003: process 1 ready
00003: process 2 running
00006: process 2 finished
00006: process 1 running
00008: process 1 finished
Process 1: Run time 5, Block time 0, Wait time 3
Process 2: Run time 3, Block time 0, Wait time 3
